dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -533,26 +520,18 @@ html, body {
 }
 
 .header_left, .header_right {
-  margin-top: 9px;
-  @media screen and (max-width: 1150px) {
-    margin-top: 5px;
-  }
+  margin-top: 5px;
 }
 
 .header_logo {
   float: left;
-  width: 86px;
-  max-width: 86px;
-  min-width: 86px;
   z-index: 1;
   margin-top: -24px;
-  @media screen and (max-width: 1150px) {
-    width: 55px;
-    max-width: 55px;
-    min-width: 55px;
-    padding-top: 14px;
-    min-width: 0px;
-  }
+  padding-top: 14px;
+  min-width: 0px;
```

Comment:
no kidding. I checked in the browser to make sure that I deleted the one that was already being ignored. It also looks like we may be ok just setting width, but I didn't want to change any more than I had to at this point.